tests/consumer_tests/routers/conftest.py

```diff
@@ -19,13 +17,13 @@ def response_row():
     This is only a subset of a response that is used in the middleware.
     """
     yield {
-        'asset_tag': 'SMP-TEST',
-        'model': {'name': 'TestName'},
-        'manufacturer': {'name': 'SmallCo'},
-        'status_label': {'status_meta': 'deployed'},
-        'location': {'name': 'Kiel'},
-        'serial': '119202',
-        'id': 1123,
+        "asset_tag": "SMP-TEST",
+        "model": {"name": "TestName"},
```

Comment:
How come the folders within `/tests/` were renamed `/{package}_tests/`?

Reply:
`mypy` has issues when package resolve with similar names. There are complicated work-arounds to this, or a more simple one is to rename to avoid this issue. **thoughts** @jawrainey?